packages/components/Molecules/Form/__snapshots__/stories.storyshot

```diff
@@ -1343,63 +1343,44 @@ exports[`RippleStoryshots Molecules/Form Default 1`] = `
                   Markup field
                 </h3>
                  
-                <figure
-                  class="rpl-document-link"
+                <article
+                  class="embedded-entity embedded-entity--media embedded-entity--media--document"
                 >
-                  <a
-                    aria-label=" Detailed Guide on the mandatory IR management criteria  File type: docx. Size: 75.22 KB"
-                    class="rpl-document-link__link"
-                    download=""
-                    href="/sites/default/files/2018-10/Detailed%20Guide%20on%20the%20mandatory%20IR%20management%20criteria.docx"
+                  <article
+                    class="media media--type-document media--view-mode-embedded"
```

Comment:
An update to markup component seemed to have re-arranged all these snapshots changes.

Reply:
@rubyllamadora it looks like the markup is plugin for document link is stop working in the PR. It should be convert to `rpl-document-link` in the story.

Reply:
So this is from the form stories here https://github.com/dpc-sdp/ripple/blob/develop/packages/components/Molecules/Form/stories.js#L356

If converted, it will become the previous `<figure class="rpl-document-link" ...`?
I thought it wasn't just the right snapshot before, I'll have a look. It's weird that having a recommended default value to props will cause this.